<map id="openai._exceptions.UnprocessableEntityError" name="openai._exceptions.UnprocessableEntityError">
<area shape="rect" id="node1" title=" " alt="" coords="5,319,231,360"/>
<area shape="rect" id="node2" href="$classopenai_1_1__exceptions_1_1APIStatusError.html" title=" " alt="" coords="19,229,217,271"/>
<area shape="rect" id="node3" href="$classopenai_1_1__exceptions_1_1APIError.html" title=" " alt="" coords="24,155,212,181"/>
<area shape="rect" id="node4" href="$classopenai_1_1__exceptions_1_1OpenAIError.html" title=" " alt="" coords="13,80,223,107"/>
<area shape="rect" id="node5" title=" " alt="" coords="78,5,158,32"/>
</map>
